sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -369,6 +369,7 @@ interface NetworkProfileDefinitionStages {
              * Docker bridge network.
              *
              * @param <ParentT> the stage of the network profile definition to return to after attaching this definition
+             * @deprecated The property has no effect since 2019
```

Comment:
move it to the API of `@Deprecated` annotation

Reply:
Is change to `@Deprecated(since = "The property has no effect since 2019")` ? This description is inconsistent with the existing description. For example:
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/23f66bfb-527b-4bb3-bc90-fc2704ee9d9b)

Reply:
Ok, fixed in the new version.